ui/de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -291,6 +295,11 @@ public void onShowAllSelected() {
     }
 
     @Override
+    public void onShareSelected(List<Analysis> selected) {
+        aSharingDialgoProvider.get().show(selected);
```

Comment:
Probably this should be "Dialog", even though it's just a variable name, rather than "Dialgo"